src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate[] dateOfCalendar = new LocalDate[data.length];
+        String[] nameOfEmployee = new String[data.length];
+        int[] workingHours = new int[data.length];
+        int[] incomePerHour = new int[data.length];
+        int[] salaryOfEmployee = new int[names.length];
```

Comment:
To avoid using these unnecessary arrays, run the second loop in the first one and use splitData[0], splitData[1], splitData[2] instead